frontend/wrangler.toml

```diff
@@ -1,13 +1,6 @@
 name = "publisher-tools-api"
-main = "functions/[[worker]].ts"
-compatibility_flags = [ "nodejs_compat" ]
+compatibility_flags = ["nodejs_compat"]
 compatibility_date = "2024-09-23"
 
 [dev]
-port = 5100
-
-[site]
-bucket = "./public"
-
-[vars]

```

Comment:
I think we want to define the non-secret vars here. `.dev.vars` will let us override during dev, but otherwise, we'll use production ones defined here.